tests/test_util/aiohttp.py

```diff
@@ -15,7 +16,7 @@ def __init__(self):
         self.mock_calls = []
 
     def request(self, method, url, *,
-                auth=None,
+                auth=None,  # pylint: disable=unused-variable
```

Comment:
`unused-argument` is present in pylintrc, but not `unused-variable`. I decided against adding `unused-variable` since it seems too wide.

According to the [documentation](http://pylint-messages.wikidot.com/messages:w0613) `unused-argument` should be raised here, so not sure why I'm getting `unused-variable` here